src/OOVPADatabase/D3D8LTCG/3911.inl

```diff
@@ -905,7 +905,7 @@ OOVPA_END;
 // * D3DDevice_SetTexture
 // ******************************************************************
 //81C10000F8FF89 ...C20400
-OOVPA_NO_XREF(D3DDevice_SetTexture_4, 2024, 9)
+OOVPA_NO_XREF(D3DDevice_SetTexture_0, 2024, 9)
 
         { 0x00, 0x83 },
         { 0x01, 0xEC },
```

Comment:
I suggest you to find where eax is set to a register or a location. Then add offset value pair(s) in here to reduce false positive.
i.e.
```
OV_MATCH(0x00, 0x83, 0xEC),
OV_MATCH(0x27, 0x81, 0xC1, 0x00, 0x00, 0xF8, 0xFF, 0x89),
```

Reply:
Done, matched on `mov esi, eax`.